### DISCUSSION TITLE
The domain of the school change

### DISCUSSION THREAD

**Discussion by louloudev59** [community/community#125702] [GitHub Education]
### Hi everyone,

Hello everyone,

When trying to register, I enter my school and my email address, but it tells me that the domain is not valid. Last year, it was .pro but recently it has changed to .fr.

I was wondering if you could update this to the domain https://www.bellecour.fr/

Thank you very much, have a great day.

**Comment by y0geshx** [comment]
Ask GitHub support via email they might help you

**Comment by louloudev59** [comment]
Thanks a lot !
